src/test/java/DetermineCourseCompletionTest.java

```diff
@@ -0,0 +1,49 @@
+import org.junit.Test;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+
+import static org.junit.Assert.*;
+
+public class DetermineCourseCompletionTest {
+
+    @Test
+    public void shouldDetermineEndDateWhenCourseEndsAt6PM() {
+        LocalDateTime startDate = LocalDateTime.of(2022, Month.FEBRUARY, 4, 10, 0);
+        int coursesDuration = 64;
+
+        LocalDateTime actual = DetermineCourseCompletion.determineEndDateSkippingWeekends(startDate, coursesDuration);
```

Comment:
Could you make the tests working against `DetermineCourseCompletion.getCourseCompletionDate` method? 
thanks